libbeat/scripts/generate_template.py

```diff
@@ -114,6 +116,16 @@ def fill_field_properties(field, defaults):
         properties[field["name"]] = {
             "type": "date"
         }
+    elif field.get("type") == "string" and field["name"] == "message":
```

Comment:
I adjusted it to your first example as a prefer to have a less "nobs" in the script and more in the config file to keep it dynamic.